controller/src/main/java/org/jboss/as/controller/AttributeDefinition.java

```diff
@@ -849,7 +860,11 @@ public ModelNode getNoTextDescription(boolean forOperation) {
             }
         }
         addAllowedValuesToDescription(result, validator);
-
+        arbitraryDescriptors.entrySet().stream().forEach((arbitraryDescriptor) -> {
+            if(!result.hasDefined(arbitraryDescriptor.getKey())) {//You can't override an arbitrary descriptor set through other properties.
```

Comment:
There should be an assert for this, as it's a programming mistake that should fail tests.